elementary/config/config.py

```diff
@@ -166,31 +162,31 @@ def has_gcs(self):
         return self.gcs_bucket_name and self.has_gcloud
 
     def validate_monitor(self):
-        self._validate_elementary_profile()
+        self._validate_internal_dbt_project()
         self._validate_timezone()
         if not self.has_slack:
             raise InvalidArgumentsError(
                 "Either a Slack token and a channel or a Slack webhook is required."
             )
 
     def validate_report(self):
-        self._validate_elementary_profile()
+        self._validate_internal_dbt_project()
 
     def validate_send_report(self):
-        self._validate_elementary_profile()
+        self._validate_internal_dbt_project()
         if not self.has_send_report_platform:
             raise InvalidArgumentsError(
                 "You must provide a platform to upload the report to (Slack token / S3 / GCS)."
             )
 
-    def _validate_elementary_profile(self):
-        profiles_path = os.path.join(self.profiles_dir, "profiles.yml")
-        try:
-            profiles_yml = OrderedYaml().load(profiles_path)
-            if "elementary" not in profiles_yml:
-                raise NoElementaryProfileError
-        except FileNotFoundError:
-            raise NoProfilesFileError(self.profiles_dir)
+    def _validate_internal_dbt_project(self):
```

Comment:
This is also not entirely necessary.
If we remove it then something else will raise `DbtCommandError`.
The problem is that on certain things we ignore the error and continue.
This is safer in the sense that if it fails, everything else is _guaranteed_ to fail as well.

Reply:
Sounds right!